doc/api/tty.md

```diff
@@ -165,6 +165,15 @@ to pass in an object with different settings.
 Use the `NODE_DISABLE_COLORS` environment variable to enforce this function to
 always return 1.
 
+To enforce a specific color support, use one of the below environment settings.
+This rule does not apply to Windows OS, CI environments, and some other rare
+cases.
+
+* 2 colors: `NODE_DISABLE_COLORS = 1`
```

Comment:
This works on Windows

Reply:
Good catch. I updated the comment again to reflect that. PTAL.